modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceDetailsViewManager.java

```diff
@@ -199,7 +213,84 @@ public List<DeployedService> listDeployedServicesOfIsv(Category category, Csp cs
         String namespace = userServiceHelper.getCurrentUserManageNamespace();
         query.setNamespace(namespace);
         List<ServiceDeploymentEntity> deployServices = serviceDeploymentStorage.listServices(query);
-        return deployServices.stream().map(EntityTransUtils::convertToDeployedService).toList();
+        return setServiceConfigurationForDeployedService(deployServices);
+    }
+
+    private void setServiceConfigurationDetailsForDeployedService(DeployedService details,
+            ServiceDeploymentEntity serviceDeployment) {
+        ServiceTemplateEntity serviceTemplate = serviceTemplateStorage
+                .getServiceTemplateById(serviceDeployment.getServiceTemplateId());
+        if (Objects.nonNull(serviceTemplate)) {
+            ServiceConfigurationManage serviceConfigurationManage =
+                    serviceTemplate.getOcl().getServiceConfigurationManage();
+            if (Objects.nonNull(serviceConfigurationManage)) {
+                List<ServiceConfigurationParameter> configurationParameters =
+                        serviceConfigurationManage.getConfigurationParameters();
+
+                Map<String, Object> configuration =
+                        Objects.nonNull(details.getServiceConfigurationDetails())
+                                ? details.getServiceConfigurationDetails().getConfiguration()
+                                : null;
+
+                ServiceConfigurationDetails serviceConfigurationDetails =
+                        getServiceConfigurationDetails(configurationParameters,
+                                configuration,
+                                serviceTemplate.getLastModifiedTime());
+                details.setServiceConfigurationDetails(serviceConfigurationDetails);
+            }
+        }
+    }
+
+    private ServiceConfigurationDetails getServiceConfigurationDetails(
+            List<ServiceConfigurationParameter> parameters, Map<String, Object> configuration,
+            OffsetDateTime updateTime) {
+        ServiceConfigurationDetails details = new ServiceConfigurationDetails();
+        Map<String, Object> configurationParameterMap = new HashMap<>();
+        parameters.forEach(configurationParameter -> {
+            configurationParameterMap.put(configurationParameter.getName(),
+                    configurationParameter.getInitialValue());
+        });
+        if (!CollectionUtils.isEmpty(configuration)) {
```

Comment:
does this also handle the 'merge' use case?
where the `configuration` is not empty but the `parameters` in the template have change (some new parameters added and some old ones removed)?